docs for httpx v0.1.0 (#1120)
grafana/k6-docs: src/data/markdown/docs/20 jslib/01 jslib/02 httpx/09 request(method, url, [body], [params]).md

Consider using specific methods for making common requests ([get](/javascript-api/jslib/httpx/get), [post](/javascript-api/jslib/httpx/post), [put](/javascript-api/jslib/httpx/put), [patch](/javascript-api/jslib/httpx/patch))

| Parameter         | Type                                                                                      | Description                                                                                 |
|-------------------|-------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------|

> | method            | string                                                                                    | HTTP method. Note, the method must be uppercase (GET, POST, PUT, PATCH, OPTIONS, HEAD, etc) |

Reviewer: ```suggestion
| method            | string                                                                                    | HTTP method. Must be uppercase (GET, POST, PUT, PATCH, OPTIONS, HEAD, etc) |
```

| url               | string                                                                                    | HTTP URL. If baseURL is set, provide only path.                                             |

> | body (optional)   | null / string / object / ArrayBuffer / [SharedArray](/javascript-api/k6-data/sharedarray) | Request body; objects will be `x-www-form-urlencoded`. Set to `null` to omit the body.      |

Reviewer: ```suggestion
| body (optional)   | null / string / object / ArrayBuffer / [SharedArray](/javascript-api/k6-data/sharedarray) | Request body; objects are `x-www-form-urlencoded`. To omit body, set to `null`.      |
```
